virtcontainers/sandbox.go

```diff
@@ -1062,6 +1063,26 @@ func (s *Sandbox) stopVM() error {
 		s.Logger().Info("Not stopping VM")
 		return nil
 	}
+	pid := s.hypervisor.pid()
+
+	defer func() {
+		// if use PhysicalEndpoint, wait for hypervisor exited
```

Comment:
missing condition?  `if PhysicalEndpoint {`

Reply:
Here I want add a explain, not condition.